VSS.NuGet.Bootstrap.nuspec

```diff
@@ -3,10 +3,15 @@
   <metadata>
     <id>Microsoft.VisualStudio.Services.NuGet.Bootstrap</id>
     <version>$version$</version>
+    <title>NuGet Bootstrapper for Microsoft Visual Studio Team Services</title>
     <authors>Microsoft</authors>
-    <owners>Microsoft</owners>
-    <requireLicenseAcceptance>false</requireLicenseAcceptance>
-    <description>Bootstrap a repository for use with Visual Studio Online Package Management</description>
+    <owners>microsoft,nugetvss</owners>
+    <licenseUrl>http://go.microsoft.com/fwlink/?LinkId=329770</licenseUrl>
+    <projectUrl>http://go.microsoft.com/fwlink/?LinkId=698609</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkId=615568</iconUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
```

Comment:
Should requireLicenseAcceptance be false? That's what we have in the AuthHelper.